lib/elixir/test/elixir/kernel/errors_test.exs

```diff
@@ -676,160 +733,169 @@ defmodule Kernel.ErrorsTest do
     after
       ;
     end
-    '''
+    ''')
 
     # All invalid examples
-    assert_eval_raise SyntaxError,
-      "nofile:1: syntax error before: ';'",
-      '1+;\n2'
+    assert_eval_raise(SyntaxError, "nofile:1: syntax error before: ';'", '1+;\n2')
 
-    assert_eval_raise SyntaxError,
-      "nofile:1: syntax error before: ';'",
-      'max(1, ;2)'
+    assert_eval_raise(SyntaxError, "nofile:1: syntax error before: ';'", 'max(1, ;2)')
   end
 
   test "new line error" do
-    assert_eval_raise SyntaxError,
+    assert_eval_raise(
+      SyntaxError,
       "nofile:3: unexpectedly reached end of line. The current expression is invalid or incomplete",
       'if true do\n  foo = [],\n  baz\nend'
+    )
   end
 
   test "characters literal are printed correctly in syntax errors" do
-    assert_eval_raise SyntaxError,
-      "nofile:1: syntax error before: \"?a\"",
-      ':ok ?a'
-    assert_eval_raise SyntaxError,
-      "nofile:1: syntax error before: \"?\\\\s\"",
-      ':ok ?\\s'
-    assert_eval_raise SyntaxError,
-      "nofile:1: syntax error before: \"?す\""
-      ':ok ?す'
+    assert_eval_raise(SyntaxError, "nofile:1: syntax error before: \"?a\"", ':ok ?a')
+    assert_eval_raise(SyntaxError, "nofile:1: syntax error before: \"?\\\\s\"", ':ok ?\\s')
+    assert_eval_raise(SyntaxError, "nofile:1: syntax error before: \"?す\"")
+    ':ok ?す'
   end
 
   test "invalid \"fn do expr end\"" do
-    assert_eval_raise SyntaxError,
+    assert_eval_raise(
+      SyntaxError,
       "nofile:1: unexpected token \"do\". Anonymous functions are written as:\n\n" <>
```

Comment:
Please extract the message to a variable on its own line.

Reply:
Like so?
```
    message =
      "nofile:1: unexpected token \"do\". Anonymous functions are written as:\n\n" <>
        "    fn pattern -> expression end\n\n" <> "Syntax error before: do"

    assert_eval_raise(SyntaxError, message, 'fn do :ok end')
```